### PR TITLE
docs/dev/libvirt/README: Minor grammar related fix

### PR DESCRIPTION
**Pull request by praveenkumar**
This PR is to test out libvirt CI. please hold it until that success.

### CODE REVIEW

File: docs/dev/libvirt/README.md [hunk 1]
```diff
@@ -153,7 +153,7 @@ sudo firewall-cmd --zone=libvirt --add-service=libvirt
 
 NOTE: When the firewall rules are no longer needed, `sudo firewall-cmd --reload`
 will remove the changes made as they were not permanently added. For persistence,
-add `--permanent` to the `firewall-cmd` commands and run them a second time.
+add `--permanent` argument to the `firewall-cmd` commands and run them a second time.
```
Comment (wking) [inline docs/dev/libvirt/README.md]: I'm fine with the previous line.  If you want to elaborate, I'd prefer "the `--permanent` option".

Comment (abhinavdahiya) [reply docs/dev/libvirt/README.md]: ping @praveenkumar

**Comment by patrickdillon** [comment]
I think this was a throwaway PR intended only to test CI. If I am mistaken, please reopen.

/close

**Pull request closed without merging**